.github/CODEOWNERS

```diff
@@ -6,10 +6,10 @@
 
 # Workspace team codeowners
 /docs/workspace/enclosure/ @jannikgeng @yunusclk # enclosure team members
-/docs/workspace/esu/ @pascalguttmann-testing # esu team members
+/docs/workspace/esu/ @mohamedshafi07 @Ranjithraj2001 @KovarthananS # esu team members
```

Comment:
`Ranjithraj2001` has not accepted collaborator invite so far. GitHub shows an error, that the user is unknown.

Think this is ok, to merge and will be resolved automatically, when collaborator invite is accepted.